app.bluebubbles.BlueBubbles.yml

```diff
@@ -0,0 +1,189 @@
+  - --share=network
+  - --filesystem=xdg-data
+  - --filesystem=xdg-download
+modules:
+  - name: bluebubbles
+    buildsystem: simple
+    build-commands:
+      - 'mkdir -p bluebubbles'
+      - 'tar -xf bluebubbles-linux-portable.tar.gz -C bluebubbles'
+      - 'cp -r bluebubbles /app/'
+      - 'chmod +x /app/bluebubbles/bluebubbles'
+      - 'mkdir -p /app/bin'
+      - 'ln -s /app/bluebubbles/bluebubbles /app/bin/bluebubbles'
+      - 'mkdir -p /app/lib'
+      - 'mv libvlc.so.5/lib64/* /app/lib/'
+      - 'mv libappindicator3.so.1/lib64/* /app/lib/'
+      - 'mv libindicator3.so.7/lib64/* /app/lib/'
+      - 'mv libdbusmenu-glib.so.4/lib64/* /app/lib/'
+      - 'mv libvlccore.so.9/lib64/* /app/lib/'
+      - 'mv libdbusmenu-gtk3.so.4/lib64/* /app/lib/'
+      - 'mv libidn.so.11/lib64/* /app/lib/'
+      - 'mv libobjectbox.so/libobjectbox.so /app/lib/'
+      - 'cp zenity/bin/zenity /app/bin/'
+      - 'mkdir -p /app/share/metainfo'
+      - 'cp -r app.bluebubbles.BlueBubbles.metainfo.xml /app/share/metainfo/'
+      - 'mkdir -p /app/share/icons/hicolor/64x64/apps'
+      - 'cp -r 64x64.png /app/share/icons/hicolor/64x64/apps/app.bluebubbles.BlueBubbles.png'
+      - 'mkdir -p /app/share/icons/hicolor/128x128/apps'
+      - 'cp -r 128x128.png /app/share/icons/hicolor/128x128/apps/app.bluebubbles.BlueBubbles.png'
+      - 'mkdir -p /app/share/icons/hicolor/256x256/apps'
+      - 'cp -r 256x256.png /app/share/icons/hicolor/256x256/apps/app.bluebubbles.BlueBubbles.png'
+      - 'mkdir -p /app/share/icons/hicolor/512x512/apps'
+      - 'cp -r 512x512.png /app/share/icons/hicolor/512x512/apps/app.bluebubbles.BlueBubbles.png'
+      - 'mkdir -p /app/share/applications'
+      - 'cp -r app.bluebubbles.BlueBubbles.desktop /app/share/applications/'
+      - 'mkdir -p /app/share/appdata'
+      - 'cp -r app.bluebubbles.BlueBubbles.metainfo.xml /app/share/appdata/'
+    sources:
+      - type: file
+        path: bluebubbles-linux-portable.tar.gz
+      - type: file
+        path: app.bluebubbles.BlueBubbles.metainfo.xml
+      - type: file
+        path: icons/64x64.png
+      - type: file
+        path: icons/128x128.png
+      - type: file
+        path: icons/256x256.png
+      - type: file
+        path: icons/512x512.png
+      - type: file
+        path: app.bluebubbles.BlueBubbles.desktop
+      - type: file
+        path: app.bluebubbles.BlueBubbles.metainfo.xml
+      - type: archive
+        only-arches:
+          - x86_64
+        url: https://ftp.lysator.liu.se/pub/opensuse/distribution/leap/15.3/repo/oss/x86_64/libvlc5-3.0.16-bp153.2.3.1.x86_64.rpm
+        archive-type: rpm
+        sha256: a684e1e2aa6821c568d1389480c752cc76b18cac8450fc6b2e0d8a8c3f170407
+        dest: libvlc.so.5
+      - type: archive
+        only-arches:
+          - x86_64
+        url: https://ftp.lysator.liu.se/pub/opensuse/distribution/leap/15.3/repo/oss/x86_64/libappindicator3-1-12.10.1+bzr20170215-1.12.x86_64.rpm
```

Comment:
this is not a SUSE distribution. libappindicator is commonly build on Flathub by various other packages. You should do the same.

And the other dependencies.